app/helpers/application_helper.rb

```diff
@@ -148,6 +148,36 @@ def bootstrap_class_for(flash_type)
     end
   end
 
+  def toast_bootstrap_flash
+    flash_messages = []
+    flash.each do |type, message|
+      type = 'success' if type == 'notice'
+      type = 'error'   if type == 'alert'
+      text = "<script>
+          toastr.options = {
+            'closeButton': false,
+            'debug': false,
+            'newestOnTop': false,
+            'progressBar': true,
+            'positionClass': 'toast-bottom-center',
+            'preventDuplicates': true,
+            'onclick': null,
+            'showDuration': '300',
+            'hideDuration': '100',
+            'timeOut': '5000',
+            'extendedTimeOut': '1000',
+            'showEasing': 'swing',
+            'hideEasing': 'linear',
+            'showMethod': 'fadeIn',
+            'hideMethod': 'fadeOut'
+          };
+          toastr.#{type}('#{message}');
+      </script>"
+      flash_messages << text.html_safe if message
```

Comment:
Don't do that! You also make use ``html_safe`` on the message! However, message can be a user input which leads to a XSS vulnerability!